src/azure-cli/azure/cli/command_modules/mysql/tests/latest/test_mysql_scenario.py

```diff
@@ -2169,3 +2171,77 @@ def _test_identity_aad_admin_mgmt(self, database_engine, resource_group, passwor
         # delete everything
         for server_name in [replica[0], replica[1], server]:
             self.cmd('{} flexible-server delete -g {} -n {} --yes'.format(database_engine, resource_group, server_name))
+
+class MySQLExportTest(ScenarioTest):
+    profile = None
+
+    def get_current_profile(self):
+        if not self.profile:
+            self.profile = self.cmd('cloud show --query profile -otsv').output
+        return self.profile
+
+    def get_account_key(self, group, name):
+        if self.get_current_profile() == '2017-03-09-profile':
+            template = 'storage account keys list -n {} -g {} --query "key1" -otsv'
+        else:
+            template = 'storage account keys list -n {} -g {} --query "[0].value" -otsv'
+
+        return self.cmd(template.format(name, group)).output
+
+    def get_account_info(self, group, name):
+        """Returns the storage account name and key in a tuple"""
+        return name, self.get_account_key(group, name)
+    
+    def storage_cmd(self, cmd, account_info, *args):
+        cmd = cmd.format(*args)
+        cmd = '{} --account-name {} --account-key {}'.format(cmd, *account_info)
+        return self.cmd(cmd)
+    
+    def create_container(self, account_info, prefix='cont', length=24):
+        container_name = self.create_random_name(prefix=prefix, length=length)
+        self.storage_cmd('storage container create -n {}', account_info, container_name)
+        return container_name
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(location=DEFAULT_LOCATION)
+    @ServerPreparer(engine_type='mysql', location=DEFAULT_LOCATION)
+    @StorageAccountPreparer(location=DEFAULT_LOCATION)
+
+    def test_mysql_export(self, resource_group, server, storage_account):
+        self._test_flexible_server_export_create_mgmt('mysql', resource_group, server, storage_account)
+
+    def _test_flexible_server_export_create_mgmt(self, database_engine, resource_group, server, storage_account):     
+        backup_name = "testexport"
+
+        target_account_info = self.get_account_info(resource_group, storage_account)
```

Comment:
```suggestion
    @AllowLargeResponse()
    @ResourceGroupPreparer(location=DEFAULT_LOCATION)
    @ServerPreparer(engine_type='mysql', location=DEFAULT_LOCATION)
    @StorageAccountPreparer(location=DEFAULT_LOCATION)

    def test_mysql_export(self, resource_group, server, storage_account_info):
        self._test_flexible_server_export_create_mgmt('mysql', resource_group, server, storage_account_info)

    def _test_flexible_server_export_create_mgmt(self, database_engine, resource_group, server, storage_account_info):     
        backup_name = "testexport"

        target_account_info = storage_account_info
```

Reply:
As discussed, moving test case to different PR